spec/services/hyrax/collections/permissions_service_spec.rb

```diff
@@ -83,12 +83,12 @@
       end
 
       it "returns true if user has any valid group" do
-        allow(user).to receive(:user_groups).and_return(['managers'])
+        allow(user).to receive_messages(groups: ['public', 'registered', 'managers'])
         expect(subject).to be true
```

Comment:
This test was changed as part of the refactor.  The changes you have here probably not needed.